cmd/topology/topology_test.go

```diff
@@ -488,3 +496,83 @@ func TestService(t *testing.T) {
 		})
 	}
 }
+
+func TestPush(t *testing.T) {
+	confFile, err := ioutil.TempFile("", "push")
+	if err != nil {
+		log.Fatal(err)
+	}
+	fmt.Fprintln(confFile, "some bytes")
```

Comment:
remove ?

Reply:
it is generating the test data for the config file load